dojo/tools/intsights/parser.py

```diff
@@ -0,0 +1,101 @@
+import json
+
+from dojo.models import Finding
+
+
+class IntSightsParser(object):
+    """
+    IntSights Threat Intelligence Feed
+    """
+
+    def get_scan_types(self):
+        return ["IntSights Report"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "IntSights Report"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "IntSights report file can be imported in JSON format."
+
+    def _parse_assets(self, assets):
+        """
+        Parses the Assets node to create a string to be used in the Description
+        """
+
+        assets_affected = ''
+
+        for entry in assets:
+            assets_affected = f'{assets_affected} Type: {entry["Type"]}, Value: {entry["Value"]},'
+
+        return assets_affected[:-1]
+
+    def get_findings(self, file, test):
+        duplicates = dict()
+
+        if not file:
+            return []
+
+        data = file.read()
+
+        try:
+            findings = json.loads(str(data, 'utf-8'))
+        except:
+            findings = json.loads(data)
```

Comment:
Keep it simple please. let `json` package do the job
```suggestion
        data = json.load(file)
```

Reply:
@damiencarol This was part of the cookie cutter boilerplate code. Should it be removed?

Reply:
yep. the cookie cutter are very new. some bloat code like this one that we used to write in the past are not welcome now. sorry for the mislead :)

Reply:
No, this is good. This is my first non-trivial commit so I want lots of review/info as to best practices.